bitnami/mongodb/templates/standalone/scripts-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if eq .Values.architecture "standalone" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mongodb.fullname" . }}-scripts
+  namespace: {{ include "mongodb.namespace" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: mongodb
+data:
+  {{- $fullname := include "mongodb.fullname" . }}
+  {{- $releaseNamespace := include "mongodb.namespace" . }} 
+  setup.sh: |-
+    #!/bin/bash
+    if [[ -n "$MONGODB_CREDENTIAL_FILE" ]] && [[ -f "$MONGODB_CREDENTIAL_FILE" ]]; then
+        while read -r line; do
+        line=$(echo "$line" | sed 's/: /=/g')
+        declare $line; 
+        done < "$MONGODB_CREDENTIAL_FILE"
+        export MONGODB_ROOT_PASSWORD="$MONGODB_ROOT_PASSWORD"
+        export MONGODB_USERNAME="$MONGODB_USERNAME"
+        export MONGODB_PASSWORD="$MONGODB_PASSWORD"
+        export MONGODB_DATABASE="$MONGODB_DATABASE"
+        export ALLOW_EMPTY_PASSWORD="$ALLOW_EMPTY_PASSWORD"
```

Comment:
What's the  use of ALLOW_EMPTY_PASSWD?

Reply:
This is used when the auth is disabled and Mongo db is supposed to allow access with empty password. Removed from the new code